usagov/usagov-2021
# USAGOV-2045 - Configuration Sync for Drupal 10.3 Upgrade

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2045

## Description


After local upgrade of the CMS, ran "drush config:export" and committed the changes.
Most of the changes are for boolean values being set to mnemonics.  However, there were some new keys in editor.editor.uswds.   There were also quite a few views which had a heading tag added to the pager section.

The specific config files are listed in the ticket.

I did not noticed anything unusual when I ran the cms locally, however some UI folks should probably take a look.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

1. The CMS editor should be exercised to see if the new settings added to editor.editor.uswds cause any issues.
2. A couple of the views with pager changes should be examined in the CMS and static site to see if any issues are observed with the pager.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
